src/providers/packet/mock_tests.rs

```diff
@@ -34,3 +35,41 @@ fn test_boot_checkin() {
     mock.assert();
     r.unwrap();
 }
+
+#[test]
+fn test_packet_attributes() {
+    let metadata = "{
```

Comment:
You can define a raw string as `let metadata = r#"YOUR_CONTENT"#;` so that you don't have to manually escape inner quotes.

Reply:
:+1: Lesson learnt..
side note: manually escaping quotes in json file is not fun :(